README.md

```diff
@@ -4,8 +4,8 @@ teamsnap/emque-consuming](https://www.codeship.io/projects/83d08620-2023-0132-2b
 # Emque::Consuming
 
 Emque Consuming is a Ruby application framework that includes everything needed
-to create and run services capable of consuming messages from a message broker
-in a Pub/sub architecture.  Messages can be produced with the
+to create and run application capable of consuming messages from a message
```

Comment:
Part of emque-consuming's value proposition (along with the routing) is that it can run as a daemon/process and parallelize requests (things you can't do with bunny or hutch by themselves). Between service or application, or perhaps it just needs to be called out better, which points in that direction better?